docs/security/definitions.mdx

```diff
@@ -0,0 +1,135 @@
+---
+sidebar_position: 2
+---
+
+# Definitions
+
+<dl>
+  <dt>Vault data</dt>
+  <dd>
+    The collection of a user's private information that they choose to store securely within Bitwarden's secure environment.
+    This typically includes highly sensitive data such as:
+
+      - **Passwords**: Credentials for various websites, applications, and services.
+      - **Usernames**: Associated usernames for accounts.
+      - **Secure Notes**: Encrypted notes containing sensitive information that the user wants to keep
+        secure.
+      - **Credit Card Information**: Payment card details like card number, expiration date, CVV, etc.
+      - **Identities**: Personal information such as names, addresses, phone numbers, and email addresses
+        that can be used to autofill forms.
+      - **Attachments**: Any files uploaded by the user to be stored securely within the vault.
+
+    Vault data may also refer to less sensitive data such as metadata:
+
+      - **Last Updated**: The last time an item was updated.
+      - **Created Date**: The date an item was created.
+
+  </dd>
+  <dt>User</dt>
+  <dd>
+    An individual who uses Bitwarden to store, manage, and access vault data.
+  </dd>
+  <dt>Client</dt>
+  <dd>
+    The software application that the User interacts with to access their vault data. This includes
+    the official Bitwarden web vault, desktop applications, mobile applications, browser extensions, and any
+    other software provided by Bitwarden that interacts with the Bitwarden server to access or manage vault data.
+  </dd>
+  <dt>Protected data</dt>
+  <dd>
+    Data stored in a format that is unreadable without additional information. Usually synonymous to
+    encrypted, but with additional expectations about how the key is stored. Encrypted data which is
+    stored together with its decryption key in plain text is not considered to be protected, even
+    thought it is encrypted.
+  </dd>
+  <dt>Communication channel</dt>
+  <dd>
+    A medium through which two or more entities, such as processes,
+applications, or systems, exchange data or messages. This can include communication between
+components on the same machine, such as inter-process communication (IPC), or over a network, such
+as between a client and a server. Common types of communication channels include sockets, APIs,
+message queues, shared memory, and HTTP connections.
+  </dd>
+  <dt>Secure channel</dt>
+  <dd>
+    A communication channel that provides confidentiality and integrity for the data transmitted
+    between two or more parties.
+
+    - **Confidentiality**: The data is unreadable to unauthorized
+    parties, typically using encryption.
+    - **Integrity**: The data cannot be altered or tampered with
```

Comment:
Consider reversing the order of "altered or tampered with" to "tampered with or altered" to avoid the sequence "with without"